library/src/lib/combobox/combobox.component.html

```diff
@@ -29,15 +29,25 @@
                  [style.maxHeight]="maxHeight">
             <ng-content></ng-content>
             <ul fd-menu-list>
-                <li fd-menu-item *ngFor="let term of displayedValues"
-                    (click)="onMenuClickHandler($event, term)"
-                    (keydown)="onMenuKeydownHandler($event, term)">
-                    <ng-container *ngIf="itemTemplate">
+                <ng-container *ngIf="!itemTemplate">
+                    <li *ngFor="let term of displayedValues; let index = index;"
+                        (click)="onMenuClickHandler(index)"
+                        (keydown)="onMenuKeydownHandler($event, index)"
+                    >
+                        <a fd-menu-item tabindex="0"
+                           [innerHTML]="term | displayFnPipe:displayFn | highlight:inputText:highlighting">
+                        </a>
+                    </li>
+                </ng-container>
+                <ng-container *ngIf="itemTemplate">
+                    <li *ngFor="let term of displayedValues; let index = index;"
+                        (click)="onMenuClickHandler(index)"
+                        (keydown)="onMenuKeydownHandler($event, index)"
+                        fd-menu-item tabindex="0"
+                    >
```

Comment:
Why did you change this? Why does there need to be two different loops in two different ng-containers?

Reply:
Can just leave the fd-menu-item on the li? This change makes the file pretty bloaty and confusing

Reply:
We can, but then the anchors won't have fd-menu-item class. So it will have blue color.

Reply:
I will add @input to class host binding, to have a control on it, then add just a class to anchor.

Reply:
Do we even need the anchors? I'm not sure why we have them, doesn't seem like a11y would require it

Reply:
HostBinding is a decent solution in the mean time

Reply:
I just changed anchors to spans in combobox.